packages/starrating/src/react/__specs__/index.spec.js

```diff
@@ -1,80 +1,57 @@
+import { render } from '@testing-library/react'
 import React from 'react'
-import { mount } from 'enzyme'
 
-import StarRating from '..'
+import StarRating from '../index.js'
 
-jest.mock('../star', () => {
-  const Star = () => null
+describe('StarRating', () => {
+  function collectStarNodes(container) {
+    const nodeList = container.querySelectorAll('span, button')
 
-  Star.appearances = {
-    empty: 'empty',
-    half: 'half',
-    full: 'full'
+    return [...nodeList].filter(node => {
+      const label = node.getAttribute('aria-label')
+      return label && label.includes('Rate')
+    })
   }
 
-  return Star
-})
-
-describe('StarRating', () => {
-  describe('prop forwarding', () => {
-    const propsToCheck = ['className', 'style']
+  it('forwards refs', () => {
+    const ref = React.createRef()
+    render(<StarRating ref={ref} />)
 
-    it.each(propsToCheck)(`should forward "%s"`, propName => {
-      const wrapper = mount(<StarRating {...{ [propName]: null }} />)
-      expect(wrapper.prop(propName)).toBeDefined()
-    })
+    expect(ref.current).not.toBeNull()
   })
 
   describe('with default props', () => {
-    let stars, wrapper
+    let container, stars
 
     beforeEach(() => {
-      wrapper = mount(<StarRating />)
-      stars = wrapper.find('Star')
-
-      expect(stars).toHaveLength(5)
-    })
-
-    it('renders stars as NOT interactive', () => {
-      stars.forEach(star => expect(star.prop('interactive')).toBe(false))
+      ;({ container } = render(<StarRating />))
+      stars = collectStarNodes(container)
     })
-  })
-
-  describe('when onChange is defined (interactive mode)', () => {
-    let stars, wrapper
 
-    beforeEach(() => {
-      wrapper = mount(<StarRating onChange={jest.fn()} />)
-      stars = wrapper.find('Star')
-
-      expect(stars).toHaveLength(5)
-    })
+    it('renders stars as NOT interactive(span NOT button)', () => {
+      expect.assertions(5)
 
-    it('renders stars as interactive', () => {
-      stars.forEach(star => expect(star.prop('interactive')).toBe(true))
+      stars.forEach(node => {
+        expect(node.tagName.toLowerCase()).toBe('span')
+      })
     })
   })
 
-  describe('when interactive but WITHOUT a value', () => {
-    let stars, wrapper
+  describe('with onChange prop', () => {
+    let container, stars
 
     beforeEach(() => {
-      wrapper = mount(<StarRating onChange={jest.fn()} />)
-      stars = wrapper.find('Star')
-
-      expect(stars).toHaveLength(5)
-    })
-
-    it('renders stars with empty appearance', () => {
-      stars.forEach(star => expect(star.prop('appearance')).toBe('empty'))
+      const handleChange = jest.fn()
+      ;({ container } = render(<StarRating onChange={handleChange} />))
```

Comment:
Ditto.  And now that I'm looking at this, this could all be helped by just making a single `it` block instead of needing to share closure scope vars with a `beforeEach` block.